scripts/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd /construct
```

Comment:
It is a little strange that you are hard coding a relative path here

Reply:
What do you propose?

Reply:
En environment variable like FEEDSTOCK_ROOT where things are copied from

Reply:
For Docker usage, this will change nothing but I agree it could be used if someone wants to run this script outside of Docker.